.travis.yml

```diff
@@ -17,8 +17,8 @@ install: source build_tools/travis/install.sh
 
 script:
   - if [ ${COVERAGE} == "true" ];
-    then nosetests --with-coverage --cover-package=skopt --with-timer --timer-top-n 10; else
-    nosetests --with-timer --timer-top-n 10;
+    then travis_wait 25 pytest --cov=skopt --durations=10; else
+    travis_wait 25 pytest --durations=10;
```

Comment:
for my education, what does `travis_wait` do? Does it time out if it runs for more than 25s?